jobs/deis_com.groovy

```diff
@@ -0,0 +1,53 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+name = 'deis.com'
```

Comment:
@slack I thought it did but tried the actual repo name of `deis.com` and no probs in Jenkins, so figured we'd go with that